stack/docker-compose.yml

```diff
@@ -0,0 +1,94 @@
+name: default-services
+
+services:
+  postgres:
+    container_name: postgres
+    image: postgres:16.0-alpine3.18
+    environment:
+      - POSTGRES_DB=postgres
+      - POSTGRES_USER=admin
+      - POSTGRES_PASSWORD=admin
```

Comment:
Why are some environment variables injected via .env file and some hardcoded? We should find a standardized way.

Reply:
it depends if the variables have to be reused or not. 
Example:
The Postgres_DB variable you linked here is not used anywhere else
The SSO_REALM on the other hand is used in several places